docs/advanced-topics/wes/cli-wes-tutorial.rst

```diff
@@ -0,0 +1,143 @@
+Launching workflows using WES and the Dockstore CLI
+===================================================
+
+The Dockstore CLI implements the Global Alliance for Genomics and Health's (`GA4GH <https://www.ga4gh.org/>`_)
+Workflow Execution Service (`WES <https://ga4gh.github.io/workflow-execution-service-schemas/docs/>`_) standard.
+The WES standard defines a format for sending workflow execution, monitoring and retrieval requests and receiving responses between a client and server.
+
+The Benefits of WES
+--------------------
+WES supports interoperability and reproducibility when launching bioinformatics workflows against execution engines, and provides a standardized
+set of API calls for gathering pertinent information regarding server information, execution run logs, and retrieving execution outputs. This
+means that as a workflow developer you can send the same workflow execution request to two different WES servers and they will both interpret and
+launch your workflow in a similar manner.
+
+The Dockstore CLI implements the WES standard, and can send your workflows as a WES request to a server given a URL and method of authorization,
+allowing quick and easy launches of Dockstore workflows. Furthermore, since the Dockstore CLI supports local workflow launches, it's easy to
+develop a workflow locally and then, once you've ironed out all the bugs, send that workflow to your WES server of choice for remote execution.
+
+and once you're ready to launch the workflow on a WES server, the command can be as simple as:
+
+   .. code:: bash
+
+        dockstore workflow wes launch --entry github.com/dockstore-testing/dockstore-whalesay2:master
+
+WES Basics
+--------------
+The WES Schema defines 6 API calls, each of which is available via the Dockstore CLI.
+
+GetServiceInfo
+^^^^^^^^^^^^^^^^^^^^^^^^
+The official documentation for this endpoint can be found here: `GetServiceInfo <https://ga4gh.github.io/workflow-execution-service-schemas/docs/#operation/GetServiceInfo>`_
+
+This endpoint describes information regarding the WES server that you are interacting with. This includes information such as which languages
+and language versions are supported (WDL, CWL, NFL, etc...), which version of WES the server is running, and the configuration settings of server itself.
```

Comment:
missing 'the' before 'server itself'.

Reply:
fixed: https://github.com/dockstore/dockstore-documentation/pull/173/commits/582d13565fc2eab1d21f57e43b5403e714fc537b